src/next/blocks/blockRepositorySettings.ts

```diff
@@ -14,7 +14,6 @@ export const blockRepositorySettings = base.createBlock({
 							allow_auto_merge: true,
 							allow_rebase_merge: false,
 							allow_squash_merge: true,
-							default_branch: "main",
```

Comment:
This is the only part of the API that can't run on an empty repository at the moment. I don't mind removing it for now to simplify things. The new default branch in repositories is `main` now for a while anyway.